lib/posix/Kconfig

```diff
@@ -1,10 +1,45 @@
 # Copyright (c) 2024 Meta
+# Copyright (c) 2025 Tenstorrent AI ULC
 #
 # SPDX-License-Identifier: Apache-2.0
 
 menu "POSIX API Support"
 
+# POSIX Subprofile Definitions
+rsource "Kconfig.profile"
+
+# Toolchain hooks for external implementations
+rsource "Kconfig.toolchain"
+
+# POSIX C Library Extensions
+# This menu is for POSIX Option Groups that do not require OS support.
+menu "POSIX C Library Extensions"
+rsource "c_lang_support_r/Kconfig"
+rsource "c_lib_ext/Kconfig"
+endmenu
+
+# POSIX System Interfaces
+menuconfig POSIX_SYSTEM_INTERFACES
+    bool "POSIX System Interfaces"
+	depends on !NATIVE_APPLICATION
```

Comment:
Something wrong with the indentation here

Reply:
Fixed!